Sat May 20 17:24:57 CEST 2006  VMiklos <vmiklos@frugalware.org>
  * pacman: typo fix in lang check
diff -rN -u old-pacman/src/pacman/pacman.c new-pacman/src/pacman/pacman.c
--- old-pacman/src/pacman/pacman.c	2006-05-20 17:29:18.000000000 +0200
+++ new-pacman/src/pacman/pacman.c	2006-05-20 17:29:18.000000000 +0200
@@ -436,7 +436,7 @@
 
 	setlocale(LC_ALL, lang);
 	// workaround for tr_TR
-	if(!strcmp(lang, "tr_TR"))
+	if(lang && !strcmp(lang, "tr_TR"))
 		setlocale(LC_CTYPE, "C");
 	bindtextdomain("pacman", "/usr/share/locale");
 	textdomain("pacman");

